### PR TITLE
[FIX] l10_ch_invoice_reports: add bindings on reports and QR name fix

### PR DESCRIPTION
**Pull request by sonhd91**
Fix missing bindings on report and fix QR report name

### CODE REVIEW

**Review by leemannd — APPROVED** [review]

**Review by didierdonze — APPROVED** [review]
Functional test ok!

**Comment by leemannd** [comment]
Hello @OCA/local-switzerland-maintainers , if there is anyone that can merge it. It would be a pleasure

File: l10n_ch_invoice_reports/data/reports.xml [hunk 1]
```diff
@@ -13,17 +13,21 @@
         <field name="print_report_name">'Invoice-ISR-%s' % object.name</field>
         <field name="attachment_use" eval="False" />
         <field name="attachment" />
+        <field name="binding_model_id" ref="account.model_account_move" />
+        <field name="binding_type">report</field>
```
Comment (yvaucher) [inline l10n_ch_invoice_reports/data/reports.xml:17]: The default value for `binding_type` is already `report` this line is not necessary.
https://github.com/odoo/odoo/blob/14.0/odoo/addons/base/models/ir_actions_report.py#L92

Comment (leemannd) [inline l10n_ch_invoice_reports/data/reports.xml:17]: Not required, but will do no harm.

https://www.odoo.com/documentation/14.0/developer/reference/addons/actions.html?highlight=binding_model_id#report-actions-ir-actions-report

File: l10n_ch_invoice_reports/data/reports.xml [hunk 2]
```diff
@@ -13,17 +13,21 @@
...
+        <field name="name">Invoice with QR</field>
         <field name="model">account.move</field>
         <field name="report_type">qweb-pdf</field>
         <field name="report_name">l10n_ch_invoice_reports.invoice_qr_report_main</field>
         <field name="report_file">l10n_ch_invoice_reports.invoice_qr_report_main</field>
         <field name="print_report_name">'Invoice-QR-%s' % object.name</field>
         <field name="attachment_use" eval="False" />
         <field name="attachment" />
+        <field name="binding_model_id" ref="account.model_account_move" />
+        <field name="binding_type">report</field>
```
Comment (leemannd) [inline l10n_ch_invoice_reports/data/reports.xml:30]: Not required, but will do no harm

**Review by yvaucher — APPROVED** [review]
binding_type is not necessary but doesn't hurt.

File: l10n_ch_invoice_reports/data/reports.xml [hunk 3]
```diff
@@ -39,5 +43,7 @@
         <field name="print_report_name">'Invoice-QR-ISR-%s' % object.name</field>
         <field name="attachment_use" eval="False" />
         <field name="attachment" />
+        <field name="binding_model_id" ref="account.model_account_move" />
+        <field name="binding_type">report</field>
```
Comment (leemannd) [inline l10n_ch_invoice_reports/data/reports.xml:47]: Not required, but will do no harm

**Pull request merged**